charts/argo-cd/templates/redis/service.yaml

```
@@ -11,7 +11,8 @@ metadata:
     {{- end }}
   {{- with .Values.redis.service.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
+  {{- range $key, $value := . }}
+    {{ $key }}: {{ $value | quote }}
```

`{{- end }}` missing
```suggestion
    {{- range $key, $value := . }}
    {{ $key }}: {{ $value | quote }}
    {{- end }}
```

---

Thanks for your patience. I'm new to this template language, and don't have a proper setup to test this here...

---

No problem, we are happy to see contributions 🙏